aptos-move/vm-genesis/src/lib.rs

```diff
@@ -1162,6 +1298,37 @@ pub struct VestingPoolsMap {
     pub period_duration_in_seconds: u64,
 }
 
+#[derive(Clone, Debug, PartialEq, PartialOrd, Ord, Eq, Serialize, Deserialize)]
+pub struct MultiSigAccountWithBalance {
+    pub owner: AccountAddress,
+    pub additional_owners: Vec<AccountAddress>,
+    pub multisig_address: AccountAddress,
+    pub num_signatures_required: u64,
+    pub metadata_keys: Vec<String>,
+    pub metadata_values: Vec<Vec<u8>>,
+    pub timeout_duration: u64,
+    pub balance: u64,
+}
+
+impl Hash for MultiSigAccountWithBalance {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        state.write(&self.multisig_address.to_vec());
```

Comment:
Same question here. I imagine you've intentionally left out the other fields, but want to make sure.

Reply:
To make sure that only instance of the an MultiSig account is supplied to the Genesis TX Encoder function, we need to hash only the MultiSig address that is derived from the group of SingleSigners. 

This is because the same set of SingleSinger accounts can create any number of MultiSig addresses they want, but for a (Owners + sequence_number) combo, there can only be one MultiSig address. So we are hashing only the `multisig_address` field here :)